### PR TITLE
WEBUI-600: support cross repo check

### PR DESCRIPTION
**Pull request by richardsd**
This one pairs with https://github.com/nuxeo/nuxeo-elements/pull/510, but the improvements in Web UI should be merged first.
Main changes:
- rename of the ondemand wf to `cross-repo`
- adds id job for remote triggering from `nuxeo-elements` cross-repo check
- update reference to `get-branch` action

### CODE REVIEW

**Review by semisse — DISMISSED** [review]

File: .github/workflows/ondemand-main.yaml [hunk 1]
```diff
@@ -156,17 +137,6 @@ jobs:
...
-        if: ${{ github.event.inputs.skip_unit_tests == 'false' && github.event.inputs.sauce_labs == 'false' }}
-        run: npm run test
-
-      - name: Web UI Unit tests (Sauce Labs)
-        if: ${{ github.event.inputs.skip_unit_tests == 'false' && github.event.inputs.sauce_labs == 'true' }}
-        env:
-          SAUCE_USERNAME: nuxeo-web-ui
-          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
-        run: npm run test
-
```
Comment (Gabez0r) [inline .github/workflows/ondemand-main.yaml]: Shouldn't we keep doing the unit tests? We might unit test an element that depends nuxeo-elements, which could break.

File: .github/workflows/ondemand-main.yaml [hunk 2]
```diff
@@ -86,9 +102,6 @@ jobs:
           npm install
           popd
 
-      - name: Lint Web UI
-        run: npm run lint
-
```
Comment (Gabez0r) [inline .github/workflows/ondemand-main.yaml]: I think we should still lint web ui. There could be changes on elements that could trigger linting errors (renaming or moving methods among methods, for example).

**Review by Gabez0r — COMMENTED** [review]
looking good! 💪

File: .github/workflows/ondemand-main.yaml (hunk 1 above)
Comment (richardsd) [reply .github/workflows/ondemand-main.yaml]: You're right, totally forgot about that use case :/ I'll add the unit tests and linting back. Thanks

File: .github/workflows/ondemand-main.yaml (hunk 2 above)
Comment (richardsd) [reply .github/workflows/ondemand-main.yaml]: ⬆️  I'll add the linting back.

**Review by semisse — DISMISSED** [review]

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**